docs/db_overview.md

```diff
@@ -3,28 +3,34 @@ id: db_overview
 title: Database Overview
 ---
 
-The database is an important component in an AEPsych project. It stores experiment metadata and also includes each message passed between the client and server, allowing previous experiments to be replayed trial-by-trial. The database is built using [SQL Alchemy](https://www.sqlalchemy.org/). The entire structure of the AEPsych Database implementation can be seen below in the following diagram:
+The database is an essential component in an AEPsych project. It stores experiment metadata and also includes each message passed between the client and server, allowing previous experiments to be replayed trial-by-trial. The database is built using [SQL Alchemy](https://www.sqlalchemy.org/). The entire structure of the AEPsych Database implementation can be seen below in the following diagram:
 
 ![DB Diagram](assets/db_diagram.png)
 
-In this diagram, the primary table is in yellow with all child tables in blue. The green-highlighted text represents the unique ID entry for that specific table. The yellow-highlighted text corresponds to a foreign key in the primary (yellow) table.
+In this diagram, the primary Table is in yellow, with all child tables in blue. The green-highlighted text represents the unique ID entry for that specific Table. The yellow-highlighted text corresponds to a foreign key in the primary (yellow) table.
 
-The database contains four tables: one primary and three secondary tables. Each table is described as follows:
+The raw_data Table is a fact table with two dimension child tables (in green). Again, the green-highlighted text represents the unique ID entry for that specific Table. The yellow-highlighted text corresponds to a foreign key in the fact (blue) table.
 
-- **master Table**: This table is the primary table of AEPsych. This table stores information about each experiment and is used to link data stored in the other tables. The columns are described as followed:
+The database contains seven tables: one primary Table with four secondary child tables. One of those is a fact table, with another two children. The tables' relationships can be seen in the following diagram:
+
+![DB Diagram](assets/db_connections.png)
+
+Each Table is described as follows:
```

Comment:
```suggestion
Each table is described as follows:
```

Reply:
@crasanders, you're right! I just updated both diagrams! ✨